beerus_core/tests/config.rs

```diff
@@ -245,12 +247,13 @@ mod tests {
     #[test]
     fn given_ethereum_network_is_invalid_when_ethereum_network_then_returns_error() {
         let config = Config {
-            ethereum_network: "invalid".to_string(),
+            ethereum_network: "mainnet2".to_string(),
```

Comment:
"invalid".to_string() is more explicit here as "mainnet2" isn't obviously a wrong network name upon reading.

Reply:
Ok!